src/hub.js

```diff
@@ -202,6 +206,14 @@ async function enterScene(mediaStream, enterInVR, janusRoomId) {
     screenEntity.setAttribute("visible", sharingScreen);
   });
 
+  document.body.addEventListener("blocked", ev => {
```

Comment:
As dicussed, would be nice to make the adapter an event emitter and wire up event listeners on that instead, but this is probably fine for now.

Reply:
👍